src/features/metadata-based-view/MetadataBasedItemList.scss

```diff
@@ -0,0 +1,46 @@
+@import '~react-virtualized/styles.css';
+@import '../../elements/common/variables';
+
+.mv-item-grid {
+    // sass-lint:disable class-name-format
+    .ReactVirtualized__Grid {
+        outline: none;
+    }
+
+    button.mv-column-header {
+        border: none;
+        color: #767676;
+        display: flex;
+        font-weight: 400;
+        justify-content: flex-start;
+        letter-spacing: .035em;
+        outline: none;
+        padding-left: 0;
+        text-transform: capitalize;
+    }
+
+    button.mv-column-header.mv-column-header-filename {
+        margin-left: 52px;
```

Comment:
Can we avoid this magic number by using positioning (flexbox, maybe) in the parent?